### ISSUE TITLE
Missing accessibility labels for buttons

### ISSUE THREAD

**Issue by mhameed** [zoff99/ToxAndroidRefImpl#236]
Thank you for an open source and a privacy respecting application.

The application is generally accessible, but a few buttons are missing accessibility labels.
These labels are read out to blind people when they interact with the application. You can test this out by enabling talkback and interacting with the app.

The buttons that are highlighted in red are those that need labels.
Could they please be added.
Tested with v1.0.92 from fdroid.

Many Thanks
![trifa-1](https://user-images.githubusercontent.com/447017/108524364-dc4c2180-72c6-11eb-9834-11687c1f9eec.jpg)
![trifa-2](https://user-images.githubusercontent.com/447017/108524420-e837e380-72c6-11eb-9990-35e5d31ccedf.jpg)
![trifa-3](https://user-images.githubusercontent.com/447017/108524454-f0901e80-72c6-11eb-9bf7-db7c99df1044.jpg)
